src/qml/MapCanvas.qml

```diff
@@ -106,7 +106,7 @@ Item {
     // stylus clicked
     TapHandler {
       enabled: !mouseAsTouchScreen
-      acceptedDevices: PointerDevice.AllDevices & ~PointerDevice.TouchScreen
+      acceptedDevices: PointerDevice.Stylus
```

Comment:
```suggestion
      acceptedDevices: PointerDevice.Stylus | DeviceType.Mouse | DeviceType.TouchPad
```
The thing is that I would like to be able to test the stylus behavior with the mouse/trackpad for debugging

Reply:
@3nids , I've tried stylus + mouse + touchpad 2 commits ago, didn't change anything. I'm removing one item at a time to see if we can go anywhere :)